src/App/Repository/PostRepository.php

```diff
@@ -389,6 +391,13 @@ public function setSearchParams(SearchData $search)
             $this->search_query->offset(intval($sorting['offset']));
         }
 
+        if ($search->getFilter('limitUnprivileged')) {
+            $isUnprivileged = !$this->postPermissions->canUserManagePosts($this->getUser());
```

Comment:
Suggest to add comment here how the post management permission is associated to bigger limits , and also that we would like to do that with API scopes in the future.

Reply:
see above